packages/voting-contract/src/lib.rs

```diff
@@ -405,18 +376,22 @@ pub fn list_votes_by_voter<Q: CustomQuery>(
     start_after: Option<u64>,
     limit: usize,
 ) -> StdResult<VoteListResponse> {
-    let start = start_after.map(Bound::exclusive);
     let voter_addr = deps.api.addr_validate(&voter)?;
-
-    let votes: StdResult<Vec<_>> = BALLOTS_BY_VOTER
-        .prefix(&voter_addr)
+    // PrimaryKey of that IndexMap is (proposal_id, voter_address) -> (u64, Addr)
+    let start = start_after.map(|m| Bound::exclusive((m, voter_addr.clone())));
```

Comment:
Another way to do this (just for fun. The current impl is better / clearer):

```suggestion
    let start = start_after.map(|proposal_id| Bound::inclusive((proposal_id+1, Addr::unchecked("")));
```

An `inclusive` bound over `proposal_id+1` is needed, because we currently don't have something like `Addr::MAX` (but we could).

This works basically because the `(proposal_id, voter)` tuple is unique (it's the pk), so, this will not miss any keys (for any given voter, he could have voted at most only once on a given proposal)